webapp/portlet/src/main/webapp/idm-users-management/components/UsersManagementList.vue

```diff
@@ -324,10 +376,26 @@ export default {
           user.lastName = user.lastName || user.lastname || '';
           if (user.lastConnexion) {
             user.lastConnexion = Number(user.lastConnexion);
+            if (user.enrollmentDate != null) {
+              user.enrollmentStatus = 'invitationAccepted';
+              user.enrollmentDetails= this.$t('UsersManagement.enrollment.invitationAccepted', {0: this.formatDate(Number(user.enrollmentDate))});
+            } else {
+              user.enrollmentStatus = 'alreadyConnected';
+              user.enrollmentDetails= this.$t('UsersManagement.enrollment.alreadyConnected');
+            }
           } else if (user.external === 'true') {
             user.lastConnexion = this.$t('UsersManagement.lastConnexion.neverConnected');
+            user.enrollmentStatus = 'cannotBeEnrolled';
+            user.enrollmentDetails= this.$t('UsersManagement.enrollment.cannotBeEnrolled');
+          }else if (user.enrollmentDate != null) {
+            user.lastConnexion = this.$t('UsersManagement.lastConnexion.invitedToJoin');
+            user.enrollmentStatus = 'reInviteToJoin';
+            user.enrollmentDetails= this.$t('UsersManagement.enrollment.reInviteToJoin', {0: this.formatDate(Number(user.enrollmentDate))});
```

Comment:
is possible to use date-format vue component to display the date ? So that you not have to format yourself the date